### ISSUE TITLE
Icon Cards

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1387]
**Block:** Cards (clickable-icon)

_Eventually - deprecate Cards & Clickable Cards_

**Open question:** 
Do we deprecate Cards (clickable, large-image)? 

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=682-2006&t=8GtTfobAhkiZdQtz-0

**User Story:** As a Web Liaison, I want the ability to present icon cards so that a site visitor can quickly identify categories to drill into a specific topic. 

**Requirements:** 
1. Author must be able to add icon, title, description, and optionally a link to make the card clickable
2. Hover state displays with filled in card, primary background color and white text + icon (TBD - need design review)
3. Default card state includes a 2px border: #C4C4C4
4. Row should display a max of 4 cards wide on desktop and a single column of cards on mobile
5. Cards should fill left -> right so if 9 cards are authored, it will displays 2 rows of 4 cards and 1 left aligned card in a third row on desktop
6. Cards should be keyboard navigable and 'active' card should display with the primary background color and white text
7. Cards should be announced by screen reader
8. https://fonts.google.com/icons --- integrate library

**Acceptance Criteria:** 
1. Given I am viewing a page with icon cards, when the page renders, then the cards default state is center aligned, with a 2px border around cards.
2. Given I am viewing a page with icon cards, when the page renders on desktop, then the cards display in rows with a max of 4 cards filling left to right.
3. Given I am viewing a page with icon cards, when the page renders on mobile, then the cards are stacked in a single column.
4. Given I am viewing a page with icon cards, when I hover a card, then the card background color fills with primary 600 color, and the icon and text on the card displays white. 
5. Given I am using my keyboard to navigate the website, when I tab to clickable cards, then they display with the filled primary background color and white text. 
6. Given I need an accessible website, when I am using a screen reader, then the card content on page is announced by a screen reader.

**Comment by sarah0arnold** [comment]
Today's site has icons with more than one color + various icon types (Background: black and white were OOTB in revize, color icons were customized)

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/icon-cards

**Status of ACs**

1. ✅ Given I am viewing a page with icon cards, when the page renders, then the cards default state is center aligned, with a 2px border around cards.

<img width="1607" height="773" alt="Image" src="https://github.com/user-attachments/assets/7265cea4-5384-49e9-a59d-4812938d6d69" />

2. ✅ Given I am viewing a page with icon cards, when the page renders on desktop, then the cards display in rows with a max of 4 cards filling left to right.

<img width="882" height="763" alt="Image" src="https://github.com/user-attachments/assets/15a264c2-0206-49a1-8a9f-cebeb20b2582" />

3. ✅ Given I am viewing a page with icon cards, when the page renders on mobile, then the cards are stacked in a single column.

<img width="560" height="821" alt="Image" src="https://github.com/user-attachments/assets/b46845ee-9098-4269-a127-c224ef444b62" />

4. ✅ Given I am viewing a page with icon cards, when I hover a card, then the card background color fills with primary 600 color, and the icon and text on the card displays white.

<img width="1349" height="847" alt="Image" src="https://github.com/user-attachments/assets/579e6f0b-3061-4981-961c-4350f56caf64" />

5. ✅ Given I am using my keyboard to navigate the website, when I tab to clickable cards, then they display with the filled primary background color and white text. **Passed but video too large, if you want to see it visit this link to download/watch https://twodegrees1.sharepoint.com/:v:/r/teams/ClarkCounty131/Shared%20Documents/General/EDS%20QA/Proof%20of%20Testing%20Videos/Icon%20Cards_Keyboard%20and%20Screenreader.mov?csf=1&web=1&e=8ikrUU**
6. ✅ Given I need an accessible website, when I am using a screen reader, then the card content on page is announced by a screen reader. **Passed but video too large, if you want to see it visit this link to download/watch https://twodegrees1.sharepoint.com/:v:/r/teams/ClarkCounty131/Shared%20Documents/General/EDS%20QA/Proof%20of%20Testing%20Videos/Icon%20Cards_Keyboard%20and%20Screenreader.mov?csf=1&web=1&e=8ikrUU**

**Note to @sarah0arnold** 
- Icon can be authored without a link and still home content for visitors and screenreaders
